tck/old-tck-selenium/ajax/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2011, 2020 Oracle and/or its affiliates. All rights reserved.
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+-->
+
+<web-app version="5.0" xmlns="https://jakarta.ee/xml/ns/jakartaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd">
```

Comment:
Should be Servlet 6.0

Reply:
I copied these from the old tck, but, considering that that ajax tests had three apps, I could only use 1 web.xml (jsf_ajax_jsresource).

1) I should rename it
2) Is updating to web.xml to 6.0 and the faces-config to 4.0 appropriate since the older tests used these definitions?

Reply:
What about the other app web.xmls/configs?

Reply:
CDI 4.0 was declared in beans.xml which implied JEE 10.

So either make sure all deployment descriptors are JEE 10 (CDI 4.0, Faces 4.0, Servlet 6.0) or make sure all deployment descriptors are JEE 9 (CDI 3.0, Faces 3.0, Servlet 5.0)/

Reply:
> What about the other app web.xmls/configs?

All deployment descriptors should preferably be in sync with each other. Originally the deployment descriptors should follow the target Faces version the test was written for. E.g. if the test is testing something which was fixed in e.g. Faces 2.3 then all DD files should be conform JEE 8.

- JEE 10: CDI 4.0 + Faces 4.0 + Servlet 6.0
- JEE 9: CDI 3.0 + Faces 3.0 + Servlet 5.0
- JEE 8: CDI 2.0 + Faces 2.3 + Servlet 4.0
- JEE 7: CDI 1.2 + Faces 2.2 + Servlet 3.1
- JEE 6: CDI 1.0 + Faces 2.0 + Servlet 3.0